site/content/ytt/docs/develop/how-to-export-schema.md

```diff
@@ -38,12 +41,14 @@ databases:
 - name: ""
   adapter: postgresql
   host: ""
+  #@schema/deprecated ""
   port: 5432
 
 #@schema/title "Additional configuration"
 #@schema/desc "Configuration for experimental/optional components; see documentation for more details."
 #@schema/examples ("Example of additional config", {"username": "default", "password": "password", "insecureFlag": True})
 #@schema/type any=True
+#@schema/deprecated ""
```

Comment:
To make this closer to what we would recommend users do, let's add a string that explains the deprecation. For example
```suggestion
#@schema/deprecated "This data value will be removed in a future release"
```